src/Instrumentation/Laravel/composer.json

```diff
@@ -9,26 +9,24 @@
   "minimum-stability": "dev",
   "prefer-stable": true,
   "require": {
-    "php": "^8.0",
+    "php": "^8.1",
     "ext-json": "*",
-    "ext-opentelemetry": "*",
```

Comment:
Changed based on [similar issue](https://github.com/open-telemetry/opentelemetry-php/issues/1191) raised for PDO instrumentation.